src/gpu/intel/ocl/tile_ops.h

```diff
@@ -196,6 +196,21 @@ DEF_BLOCK2D_LOAD_STORE(half, ushort, 16, 16, u16_m8k32v1, 32, 8)
 #define xlane_tile_access(t, i, j, sg, br, bc, nbr) \
     sub_group_broadcast(tile_access(t, i, j, sg, br, bc, nbr), i % sg)
 
+#define tile_predicated_assignment_t( \
```

Comment:
Usually `_t` is for class/struct types, not for macros. Does it suppose to add some useful information?

Reply:
the `_t` implies transpose because the S_tile is transposed in memory. This is inline with the tile_load_t function names.

Reply:
I see, thanks for clarifying.